sdcm/nemesis.py

```diff
@@ -713,6 +766,29 @@ def modify_table_bloom_filter_fp_chance(self):  # pylint: disable=invalid-name
         """
         self._modify_table_property(name="bloom_filter_fp_chance", val=random.random() / 2)
 
+    def toggle_table_ics(self):
+        """
+            Alters a non-system table compaction strategy from ICS to any-other and vise versa.
+        """
+        # TODO: Sub-properties for each of compaction strategies should also be tested
```

Comment:
so do it.... randomize the values when changing to ICS.

Reply:
done